metpy/io/upperair.py

```diff
@@ -17,11 +17,14 @@
 from ._tools import UnitLinker
 from .cdm import Dataset
 from ..calc import get_wind_components
+from ..deprecation import deprecated
 from ..package_tools import Exporter
 
 exporter = Exporter(globals())
 
 
+@deprecated('0.8', addendum=' This function is being moved to the Siphon package.',
+            pending=True)
```

Comment:
Then I'd say change the version to '0.6' and leave it as not pending since it's not actually pending--it's done.